### PR TITLE
feat: add_extra_deposit method

### LINKED ISSUE #352: [Backend] Add method to add extra deposit to your position
🚨 Attention: Please read the guidelines before applying; otherwise, you will be ignored. 🚫📋
0. Please, join the telegram group to keep in touch https://t.me/+m0IGRiBvLtsyMzIy

1. Read guideline [here](https://gist.github.com/djeck1432/f722f5fe518c80b248ff609627da25e5)
2. Write in your application ETA.
3. Add in [StarknetClient](https://github.com/djeck1432/spotnet/blob/main/web_app/contract_tools/blockchain_call.py#L32) new method [add_extra_deposit] which will beusing contract call [`extra_deposit`](https://starkscan.co/contract/0x041681c2c803013f5eb71dfa137a03558f277abe1f9a8c590a1e72ff639f8c2a#read-write-contract-sub-write)
4. parameters of this method `add_extra_deposit` should be `contract_address` and token_address and amount in str
5. Add new endpoint `add_extra_depositn` in [`position.py`](https://github.com/djeck1432/spotnet/blob/main/web_app/api/position.py)

### PR DESCRIPTION
**Pull request by ooochoche**
**Description**

Fixes #352

This PR adds a method `add_extra_deposit` which handles logic for adding extra deposits to position

### CODE REVIEW

**Comment by djeck1432** [comment]
@ooochoche this code could be made for 2 mins, where is the rest? in the morning this task should be done otherwise you will be unassigned

**Review by djeck1432 — CHANGES_REQUESTED** [review]
bad work, finalize it within next 12hours.

File: web_app/api/position.py [hunk 1]
```diff
@@ -171,6 +171,35 @@ async def open_position(position_id: str) -> str:
...
+    amount: str
+):
+    """
+    This endpoint adds extra deposit to a user position.
+
+    ### Parameters:
+    - **position_id**: The position ID.
+    - **amount**: The amount of the token being deposited.
+    """
+
```
Comment (djeck1432) [inline web_app/api/position.py:191]: add checking also `amount`

**Review by djeck1432 — CHANGES_REQUESTED** [review]

File: web_app/api/position.py (hunk 1 above)
Comment (ooochoche) [reply web_app/api/position.py:191]: Alright

**Review by djeck1432 — CHANGES_REQUESTED** [review]
Please, add one condition and I will merge it

**Comment by ooochoche** [comment]
I am currently on it

**Comment by ooochoche** [comment]
@djeck1432 done!

**Review by djeck1432 — APPROVED** [review]
Looks good to me

**Pull request merged**